File: curso POO uber/Java/UberVan.java
import java.util.Map;
import java.util.ArrayList;

class UberVan extends Car{
    Map<String, Map<Integer, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    Integer passengers;

    // public UberVan(String license, Account driver, 
    // Map<String,Map<Integer, Integer>> typeCarAccepted,
    //  ArrayList<String> seatsMaterial){

    //     super(license, driver);
    //     this.typeCarAccepted = typeCarAccepted; 
    //     this.seatsMaterial = seatsMaterial;
    // }

    public UberVan(String license, Account driver){

        super(license, driver);
    }
    
    @Override
    public void setPassenger(Integer passengers) {
        if(passengers == 6){
            this.passengers = passengers;
        }else{
            System.out.println("Necesitas asignar 6 pasajeros");
         } 
    }
}

File: curso POO uber/Python/UberVan.py
from car import Car

class UberVan(Car):

    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, lincense, driver, typeCarAccepted, seatsMaterial):
        super().__init__(lincense, driver)

        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial  